libs/shared/components/src/lib/header/header.tsx

```diff
@@ -23,13 +23,17 @@ export function Header(props: HeaderProps) {
   window.addEventListener('scroll', () => {
```

Comment:
I think it's better to use something like `window.onscroll` instead of adding eventlisteners every time the page rerenders...Press F5 20 times and the page will be laggy af

Reply:
in that case you can only have a single scrollevent though

Reply:
why?

Reply:
![image](https://user-images.githubusercontent.com/54234287/130656645-cf2831d3-dcfd-403e-a5cd-1871d4d48dc0.png)
Have a look at it in the DevTools

Reply:
Surprisingly it doesn't lag

Reply:
if I use `window.addEventlistener("scroll", () => {})` I can add as many as I want, if I use `window.onscroll = () => {}`, I can only ever overwrite this value

Reply:
You can use `window.onscroll` a single time outside a component.

Reply:
Just saying

Reply:
should I changes this or not?

Reply:
Honestly. Yes, I think it would be nice if you could change it :D